kubernetes/filters.go

```diff
@@ -240,11 +240,24 @@ func FilterRegistryServicesByServices(registryServices []*RegistryService, servi
 	return filtered
 }
 
-func FilterByRegistryService(hostname string, registryService *RegistryService) bool {
-	// Basic filter using Hostname
-	// TODO use the ExportTo, Namespace, ServiceRegistry and other attributes to refine the match
+func FilterByRegistryService(itemNamespace string, hostname string, registryService *RegistryService) bool {
```

Comment:
Minor, in the kubernetes/filters.go it's used the "namespace string" as part of the signature, perhaps we can keep that here as well, to avoid to diverge too much in the coding style.

```
func FilterByHost(host, serviceName, namespace string) bool
func FilterDestinationRulesByService(allDr []networking_v1alpha3.DestinationRule, namespace string, serviceName string) []networking_v1alpha3.DestinationRule
func FilterVirtualServicesByService(allVs []networking_v1alpha3.VirtualService, namespace string, serviceName string) []networking_v1alpha3.VirtualService {
```

Reply:
+1